staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc_test.go

```diff
@@ -1396,148 +1388,6 @@ func TestToken(t *testing.T) {
 				Name: "thomas.jefferson@gmail.com",
 			},
 		},
-		{
-			name: "good token with api req audience",
```

Comment:
Can you make sure we have a test for incorrect client ID in the aud claim?

Reply:
Done.